public/ACMW/css/events.css

```diff
@@ -145,11 +108,19 @@ h1 span {
 	color: #1b80ce;
 	background-color: #fcfcfa;
 }
-
+footer .container{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The properties padding-top, padding-bottom, padding-left, padding-right can be replaced by padding.](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)